spikeextractors/extractors/combinatosortingextractor/combinatosortingextractor.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+import numpy as np
+from spikeextractors import SortingExtractor
+from spikeextractors.extraction_tools import check_valid_unit_id
+from typing import Union
+
+
+try:
+    import h5py
+    HAVE_H5PY = True
+except ImportError:
+    HAVE_H5PY = False
+
+PathType = Union[str, Path]
+
+
+class CombinatoSortingExtractor(SortingExtractor):
+    extractor_name = 'CombinatoSortingExtractor'
+    installation_mesg = ""  # error message when not installed
+    installed = HAVE_H5PY
+    is_writable = False
+    installation_mesg = "To use the CombinatoSortingExtractor install h5py: \n\n pip install h5py\n\n"  # error message when not installed
+
+    def __init__(self, datapath: PathType, sr=None, user='simple',det_sign = 'both'):
```

Comment:
```suggestion
    def __init__(self, datapath: PathType, sampling_frequency=None, user='simple', det_sign = 'both'):
```

Reply:
@ferchaure can you call `sr`-->`sampling_frequency` to be in line with SI semantics? Thanks :)